.github/pull_request_template.md

```diff
@@ -0,0 +1,37 @@
+# Description
+
+<!--
+  A summary of the change, including any relevant motivation and context.
+
+  If relevant, include a description both of the existing behavior and of the new behavior.
+-->
+
+## Links
+
+<!--
+  Any relevant links to external resources; ie, specification documents, jira
+  items, related PRs, honeybadger errors, etc
+-->
+
+- [spec]()
+- [jira]()
+
+## Testing story
+
+<!--
+  Does your change include appropriate tests?
+
+  If so, please describe how the tests included in this PR are sufficient
+
+  If not, please explain why this change does not need to be tested.
+-->
+
+# Reviewer Checklist:
+
+- [ ] Tests are passing and seem appropriate
```

Comment:
how about "tests provide adequate coverage"? we already have a different check mark to tell us whether tests are passing.